package.json

```diff
@@ -1,25 +1,32 @@
 {
```

Comment:
can you remove all changes to this file (please force push) ... we want to be able to create a PR on Unit's repository using this branch and we don't want to propose them change their package scope :)